routes/delete_medication/schema.js

```diff
@@ -0,0 +1,14 @@
+/* istanbul ignore file */
+
+const Schema = {
+  firstname: {
+    isLength: {
+      errorMessage: 'errors.firstname.length',
+      options: { min: 3, max: 200 },
```

Comment:
First name minimum length should be something like 1.

We might want to consider at some point coming up with some baselines for validations for names and extracting it to a method like: 

```javascript
function firstNameLengthValidation(errMsgKey) {
  return {
    isLength: {
      errorMessage: errMsgKey,
      options: { min: 1, max: 200 },
    },
  };
} 
```

Reply:
That schema should have been deleted. it is now deleted.